### DISCUSSION TITLE
Database download speed

### DISCUSSION THREAD

**Discussion by DOH-JDJ0303** [oschwengers/bakta#277] [Polls]
My estimated download time for the light db (~1.4 GB) is 30-40 mins. Not sure if this is a me issue or if others have experienced slow download speeds as well.

**Comment by oschwengers** [comment]
Hi thanks for bringing this up. The Bakta db is stored at Zenodo which is hosted by the CERN in Swiss. Hence, download rates might be a bit low from time to time depending on the general bandwidth. Also, CERN itself sees some short-term download limits from time to time.
